README.md

```
@@ -1,396 +1,292 @@
+- [Bora Rachar](#bora-rachar)
+- [Tem Vaga Mestre?](#tem-vaga-mestre)
+- [DeliveryAPP](#deliveryapp)
+- [GoLembrar](#golembrar)
+- [Amigos dos Animais](#amigos-dos-animais)
+- [Docunder](#docunder)
+- [Gym-progress](#gym-progress)
+- [QuickServices](#quickservices)
+- [Hello-Pets](#hello-pets)
+- [Diaristando](#diaristando)
+- [SoliciteMe!](#solicite-me)
+- [Diagier](#Diagier)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #diagier; Actual: #Diagier

(MD051, link-fragments)